openedx/core/djangoapps/programs/utils.py

```diff
@@ -332,6 +333,7 @@ def __init__(self, program_data, user):
     def extend(self):
         """Execute extension handlers, returning the extended data."""
         self._execute('_extend')
+        self.data['user_preferences'] = get_user_preferences(self.user)
```

Comment:
This is not program-specific data, so it shouldn't be added here. If it's necessary on the page, add it separately to the [template context](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/learner_dashboard/views.py#L72).